roslibrust/tests/ros1_xmlrpc.rs

```diff
@@ -52,23 +44,19 @@ mod tests {
             vec!["/get_master_uri_test".into()],
         )
         .await;
+        log::info!("Got master");
         assert_eq!(master_uri, "http://localhost:11311");
+        Ok(())
     }
 
-    #[tokio::test]
-    async fn verify_get_publications() {
-        let node = timeout(
-            TIMEOUT,
-            roslibrust::NodeHandle::new("http://localhost:11311", "verify_get_publications"),
-        )
-        .await
-        .unwrap()
-        .unwrap();
+    #[test_log::test(tokio::test(flavor = "multi_thread"))]
```

Comment:
Does this still require the multi thread flavor and the test_log macro?

Reply:
1. I think all of our tests should be wrapped in test_log

2. Maybe, have tried a lot of permutations of testing, but it is hard to get the failures to manifest... I don't think it is harmful being in there for now.